src/main/java/com/marklogic/contentpump/Command.java

```diff
@@ -1993,7 +1993,6 @@ protected static boolean isStreaming(CommandLine cmdline,
      *            Hadoop configuration
      * @param cmdline command line options
      * @return a Hadoop job
-     * @throws Exception
```

Comment:
This should be throws IOException, right?

Reply:
Yes, it should be throws IOException instead of deleting